stable/rails/values.yaml

```diff
@@ -25,12 +25,25 @@ migrations:
   - "bundle exec rake db:migrate"
 
 ingress:
-  enabled: false
-  host: myhost.local
-  annotations: {}
-  tls:
-    enabled: false
-    secret: myhost.local-tls
+  enabled: true
+  list:
+    - host: myhost1.local
+      type: public
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: 256m
+        nginx.ingress.kubernetes.io/server-snippet: |
```

Comment:
That's a good option that gives us the ultimate flexibility. 

If most of the time we will be denying certain path how about following syntax to make it simpler for configuration:
```
- host: myhost
   deny_locations:
     - /admin
     - /internal_api
     - /etc
```
That will be transformed into a server snippet. 
If a person specified server snippet and deny_locations - server snippet takes priority.